docs/diagrams/InsertClassDiagram.puml

```diff
@@ -23,28 +12,10 @@ interface Model <<interface>> {
     + hasOrder(Order order) : boolean
 }
 
-class ArgumentMultimap {
-    + getValue(prefix : Prefix) : Optional<String>
-}
+abstract class "{abstract}\nParcel" {
 
-class Prefix {
-    - prefix : String
 }
 
-abstract class "{abstract}\nParcel" {
-    + getTid() : TransactionId
-    + getName() : Name
-    + getPhone() : Phone
-    + getEmail() : Email
-    + getAddress() : Address
-    + getTimestamp() : Timestamp
-    + getWarehouse() : Warehouse
-    + getCash() : CashOnDelivery
-    + getComment() : Comment
-    + getItemType() : TypeOfItem
-    + isDelivered() : boolean
```

Comment:
This is not getter or setter, hence I think Parcel should be inside.

Reply:
So you are saying I just add in isDelivered() method in Parcel? Since its the only method that is not getter or setter.

Reply:
Yeap. So I only add the isDelivered() method in the Parcel class since its the only non getter and setter method. Hopefully thats what you wanted